payments/templates/credit_card_page.html

```diff
@@ -2,7 +2,7 @@
 {% block content %}
     <div class="row">
         <div class="column medium-4 medium-offset-4 text-left">
-            {% if request.user.stripe_bank_account %}
+            {% if request.user.stripe_customer %}
                 You have already registered a credit card with us.  You can use this form to change the card information.
```

Comment:
Take out the "already" - it's confusing when you land on this template right after adding the bank account or credit card.